src/Console/Command/FixCommand.php

```diff
@@ -178,10 +179,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ('none' === $progressType || null === $stdErr) {
             $progressOutput = new NullOutput();
         } elseif ('run-in' === $progressType) {
```

Comment:
Are you sure we want to do that? `run-in` is the only option that allows to have some progress output without first iterating all files.